### PR TITLE
Add d language schema

### PR DESCRIPTION
**Pull request by GreenBayRules**
data/Language/D.yaml >> Add language schema and boolean yaml

### CODE REVIEW

File: data/Language/D.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+ - continue
+ - creal
+ - dchar
+ - debug
+ - default
+ - delegate
+ - delete
+ - deprecated
+ - do
+ - double
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ - cast
+ - catch
+ - cdouble
+ - cent
+ - cfloat
+ - char
+ - class
+ - const
+ - continue
+ - creal
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+ - assert
+ - auto
+ - body
+ - bool
+ - break
+ - byte
+ - case
+ - cast
+ - catch
+ - cdouble
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+ - cent
+ - cfloat
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+ - char
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+  - boolean_d
+keywords:
+ - abstract
+ - alias
+ - align
+ - asm
+ - assert
+ - auto
+ - body
+ - bool
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+  - double_quote_slash_escape
+datatypes:
+  - boolean_d
+keywords:
+ - abstract
+ - alias
+ - align
+ - asm
+ - assert
+ - auto
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+ - catch
+ - cdouble
+ - cent
+ - cfloat
+ - char
+ - class
+ - const
+ - continue
+ - creal
+ - dchar
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+ - deprecated
+ - do
+ - double
+ - else
+ - enum
+ - export
+ - extern
+ - final
+ - finally
+ - float
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+ - for
+ - foreach
+ - foreach_reverse
+ - function
+ - goto
+ - idouble
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+ - if
+ - ifloat
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+ - immutable
+ - import
+ - in
+ - inout
+ - int
+ - interface
+ - invariant
+ - ireal
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+ - is
+ - lazy
+ - long
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+ - 'null'
+ - out
+ - override
+ - package
+ - pragma
+ - private
+ - protected
+ - public
+ - pure
+ - real
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 15]
```diff
@@ -0,0 +1,126 @@
...
+ - switch
+ - synchronized
+ - template
+ - this
+ - throw
+ - try
+ - typedef
+ - typeid
+ - typeof
+ - ubyte
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 16]
```diff
@@ -0,0 +1,126 @@
...
+ - ucent
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+ - uint
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+ - ulong
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 19]
```diff
@@ -0,0 +1,126 @@
...
+ - union
+ - unittest
+ - ushort
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 20]
```diff
@@ -0,0 +1,126 @@
...
+ - version
+ - void
+ - wchar
```
Comment (jayvdb) [inline data/Language/D.yaml]: this is a datatype.

File: data/Language/D.yaml [hunk 21]
```diff
@@ -0,0 +1,126 @@
...
+ - __FILE__
+ - __FILE_FULL_PATH__
+ - __MODULE__
+ - __LINE__
+ - __FUNCTION__
+ - __PRETTY_FUNCTION__
+ - __gshared
+ - __traits
+ - __vector
+ - __parameters
```
Comment (jayvdb) [inline data/Language/D.yaml]: EOL at EOF

**Comment by jayvdb** [comment]
Please do not close a pull request.

This is your pull request.  There are many like it, but this one is YOURS! :P

**Comment by GreenBayRules** [comment]
shoot i think i screwed it up

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/36/commits/a0e591c0a8e175a5d3942d928a28a2c6fffa54b9

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**